<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1626</width>
    <height>947</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QMainWindow,QPushButton, QLabel, QLineEdit{
    background-color: white;
    font-size: 24px;
    font-family: &quot;华文中宋&quot;;
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btn_project">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 项目管理</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/项目管理.png</normaloff>icons/项目管理.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_condition">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 条件输入</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/表单组件-输入框.png</normaloff>icons/表单组件-输入框.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_material">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 元件定义</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/元件定义.png</normaloff>icons/元件定义.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pipe">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 管口及附件设计</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/管口.png</normaloff>icons/管口.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_pipeDesign">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 管束设计</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/建筑-管子.png</normaloff>icons/建筑-管子.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_cal">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 设计运算</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/计算.png</normaloff>icons/计算.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_2D">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 图纸绘制</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/图纸 (2).png</normaloff>icons/图纸 (2).png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_docs">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 文本说明生成</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/三维建模.png</normaloff>icons/三维建模.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_3D">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 模型创建</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/3D建模.png</normaloff>icons/3D建模.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>48</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">background: transparent;
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_login">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>65</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    padding-left: 5px;
}
</string>
            </property>
            <property name="text">
             <string> 登录/注册</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/用户 (1).png</normaloff>icons/用户 (1).png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;Arial&quot;;</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string/>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icons/mainpage.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_tip">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1626</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Microsoft YaHei&quot;;</string>
   </property>
   <widget class="QMenu" name="document">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_6"/>
    <addaction name="action_14"/>
    <addaction name="action_19"/>
   </widget>
   <widget class="QMenu" name="logo">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/logo.png</normaloff>icons/logo.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="design_mode">
    <property name="title">
     <string>设计模式</string>
    </property>
    <addaction name="action_8"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="config">
    <property name="title">
     <string>配置</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>偏好设置</string>
     </property>
     <addaction name="action_15"/>
     <addaction name="action_16"/>
     <addaction name="action_17"/>
    </widget>
    <addaction name="action_18"/>
    <addaction name="menu"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_10"/>
    <addaction name="action_13"/>
    <addaction name="action_12"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>审核</string>
    </property>
    <addaction name="action_20"/>
    <addaction name="action_21"/>
    <addaction name="action_22"/>
    <addaction name="action_23"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>输出</string>
    </property>
    <addaction name="scheme_design"/>
    <addaction name="detailed_deisign"/>
    <addaction name="action_26"/>
   </widget>
   <widget class="QMenu" name="menu_AI">
    <property name="title">
     <string>AI+</string>
    </property>
    <addaction name="action_38"/>
    <addaction name="action_39"/>
    <addaction name="action_40"/>
    <addaction name="action_41"/>
    <addaction name="action_25"/>
   </widget>
   <widget class="QMenu" name="redo">
    <property name="title">
     <string>12</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/重做.png</normaloff>icons/重做.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="revoke">
    <property name="title">
     <string>1</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/撤销.png</normaloff>icons/撤销.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="new_built">
    <property name="title">
     <string>3</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/新建.png</normaloff>icons/新建.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="open_">
    <property name="title">
     <string>4</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/打开.png</normaloff>icons/打开.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="save">
    <property name="title">
     <string>5</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/保存.png</normaloff>icons/保存.png</iconset>
    </property>
   </widget>
   <widget class="QMenu" name="save_as">
    <property name="title">
     <string>6</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/另存为.png</normaloff>icons/另存为.png</iconset>
    </property>
   </widget>
   <addaction name="logo"/>
   <addaction name="new_built"/>
   <addaction name="open_"/>
   <addaction name="save"/>
   <addaction name="save_as"/>
   <addaction name="revoke"/>
   <addaction name="redo"/>
   <addaction name="document"/>
   <addaction name="design_mode"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="config"/>
   <addaction name="menu_AI"/>
   <addaction name="help"/>
  </widget>
  <action name="actionyonghuxinxi">
   <property name="text">
    <string>用户信息</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>新建项目</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>保存项目</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>智动设计</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>灵活设计</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>查看软件使用文档</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>故障排查</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>在线支持</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>本地离线帮助手册</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>界面</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>快捷键</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>存储路径</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>预定义</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>条件评审</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>方案评审</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>技术评审</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>结果评审</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>文件存档</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>二维图纸</string>
   </property>
  </action>
  <action name="action_28">
   <property name="text">
    <string>三维模型</string>
   </property>
  </action>
  <action name="action_29">
   <property name="text">
    <string>强度计算书</string>
   </property>
  </action>
  <action name="action_30">
   <property name="text">
    <string>材料清单</string>
   </property>
  </action>
  <action name="action_31">
   <property name="text">
    <string>一键全选</string>
   </property>
  </action>
  <action name="action_32">
   <property name="text">
    <string>二维图纸</string>
   </property>
  </action>
  <action name="action_33">
   <property name="text">
    <string>三维模型</string>
   </property>
  </action>
  <action name="action_34">
   <property name="text">
    <string>强度计算书</string>
   </property>
  </action>
  <action name="action_35">
   <property name="text">
    <string>材料清单</string>
   </property>
  </action>
  <action name="action_36">
   <property name="text">
    <string>一键全选</string>
   </property>
  </action>
  <action name="action_37">
   <property name="text">
    <string>实时翻译</string>
   </property>
  </action>
  <action name="action_38">
   <property name="text">
    <string>专业术语校对</string>
   </property>
  </action>
  <action name="action_39">
   <property name="text">
    <string>数据处理</string>
   </property>
  </action>
  <action name="action_40">
   <property name="text">
    <string>审图</string>
   </property>
  </action>
  <action name="action_41">
   <property name="text">
    <string>智能提示</string>
   </property>
  </action>
  <action name="scheme_design">
   <property name="text">
    <string>方案设计</string>
   </property>
  </action>
  <action name="detailed_deisign">
   <property name="text">
    <string>详细设计</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>实时翻译</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>文件存档</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
